sklearn/model_selection/_split.py

```diff
@@ -675,9 +675,8 @@ class TimeSeriesCV(_BaseKFold):
 
     Notes
     -----
-    The first fold has size
-    ``n_samples // (n_splits + 1) + n_samples % (n_splits + 1)``,
-    and the other folds all have size ``n_samples // (n_splits + 1)``.
+	The training set has size ``i * n_samples // (n_splits + 1) + n_samples % (n_splits + 1)``
```

Comment:
You've used a tab character here. And gone over 79 characters. You can break lines within double-backticks if necessary.